src/reference/config/glossary.md

```diff
@@ -0,0 +1,159 @@
+
+```toml
+# Ed25519 public key hash
+ed25519_pub = "0xed...<hash value>"
+
+# Secp256k1 public key hash
+secp256k1_pub = "0xe7...<hash value>"
+
+# BLS12-381 G1 public key hash
+bls12381_g1_pub = "0xea...<hash value>"
+
+# BLS12-381 G2 public key hash
+bls12381_g2_pub = "0xeb...<hash value>"
+```
+
+In these examples, replace `<hash value>` with the actual hash output encoded in hexadecimal. The code before the hash value corresponds to the hash function used, as per the Iroha 2 supported options.
+
+For further details on Multihash and its implementation, please refer to the [Multihash specification](https://multiformats.io/multihash/) and the [Multicodec Table](https://github.com/multiformats/multicodec/blob/master/table.csv) for a comprehensive list of hash function codes.
+
+
+## Type: Bytes Amount
+
+Bytes amount is specified as a human-readable string:
+
+```toml
+# 42 bytes
+value1 = "42B"
+
+# 1 kilobyte = 1000 bytes
+value2 = "1KB"
+
+# 1 kilobyte (binary format) = 1024 bytes
+value3 = "1KiB"
+
+# Sum of multiple
+value4 = "1GB 5MB"
+```
+
+Iroha can parse sizes in bytes, kilobytes (`K`), megabytes (`M`), gigabytes (`G`), terabytes (`T`),
+and petabytes (`P`).
+
+The format of suffixes:
+
+- **`{size}iB`:** Binary size
+- **`{size}B`:** Decimal size
+
+TODO: haven't found a rust lib for that. There is a Python one
+([`humanfriendly`](https://humanfriendly.readthedocs.io/en/latest/api.html?highlight=parse_size#humanfriendly.parse_size))
+I used for reference.
+
+TODO: [article explains kb, kB, KiB difference](https://web.archive.org/web/20150324153922/https://pacoup.com/2009/05/26/kb-kb-kib-whats-up-with-that/).
+Consider it for the format.
+
+## Type: Private Key
+
+TODO explain private key
+
+```toml
+private_key = { algorithm = "ed25519", payload = "" }
+```
+
+## Type: Socket Address
+
+To specify a socket address in the configuration, use the format `host:port`, where `host` is either a hostname or an IP address, and `port` is the numeric port number the application should connect to or listen on.
+
+- **Hostname example**: `localhost:8080`
+- **IP address example**: `192.168.1.100:8080`
+
+**Hostname**: Can be a local hostname (`localhost` for the local machine) or a remote server's domain name (e.g., `example.com`).
+
+**IP address**: Use IPv4 (e.g., `192.168.1.100`) or IPv6 (enclosed in square brackets, e.g., `[2001:db8::1]`) formats.
+
+**Port**: A numeric value typically ranging from `1` to `65535`, where ports below `1024` are reserved for well-known services and require root privileges on Unix-like systems.
+
+**Example in TOML:**
+
+```toml
+address = "localhost:8080"
+```
+
+For more information on socket addresses and networking, refer to the [IANA Service Name and Transport Protocol Port Number Registry](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml) or the [RFC 3986](https://tools.ietf.org/html/rfc3986) specification.
+
+## Paths Resolution
```

Comment:
```suggestion
## Path Resolution
```

that affects links

Reply:
Actually, we can use [custom anchors](https://vitepress.dev/guide/markdown#custom-anchors):

```md
# Using custom anchors {#my-anchor}
```